src/util.js

```diff
@@ -0,0 +1,9 @@
+
+function isValidUUID(input) {
+    let regex = new RegExp(/^[0-9a-f]{8}-[0-9a-f]{4}-[0-5][0-9a-f]{3}-[089ab][0-9a-f]{3}-[0-9a-f]{12}$/, "i");
+    return regex.test(input);
+};
+
+module.exports = {
+    isValidUUID
+};
```

Comment:
I generally think files like `util` are probably a bad idea: they are a messy collection of lots of unrelated code. I think a (slightly) better approach would be an `utils` folder with a file for each function (in this case, `utils/isValidUuid.js`).